components/grid/toolbar.md

```diff
@@ -23,8 +23,15 @@ The [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) provides se
 | Tool Name | Tool Tag | Description |
 | --- | --- | --- |
 | Add | `GridToolBarAddTool` | An add command that fires the [`OnAdd` event](slug:grid-editing-overview#events). |
+| Cancel || `GridToolBarCancelEditTool` | Cancels the changes for the selected row. Row selection and `Inline` or `Popup` editing mode are required. |
```

Comment:
Don't leave cells blank. Enter "NA" in the cell if there is no Tool Tag. This comment applies to all blank cells.